MobalMobal/MobalMobal/Signup/SignupViewController.swift

```diff
@@ -0,0 +1,172 @@
+    
+    // MARK: - Method
+    private func setup() {
+        self.view.backgroundColor = .backgroundColor
+        
+        setUIViewLayout()
+    }
+    
+    @objc
+    func agreementButtonIsTapped(button: UIButton) {
+        agreementButtonState.toggle()
+        
+        if agreementButtonState {
+            button.setImage(UIImage(named: "iconlyLightCheckOff"), for: .normal)
+        } else {
+            button.setImage(UIImage(named: "iconlyLightCheckOn"), for: .normal)
+        }
+    }
+    
+    private func setNicknameView() {
+        self.view.addSubview(nickNameView)
+        self.nickNameView.snp.makeConstraints { make in
+            make.height.equalTo(60)
+            make.top.equalToSuperview().inset(59)
+            make.leading.trailing.equalToSuperview().inset(15)
+        }
+    }
+    
+    private func setPhoneNumberView() {
+        self.view.addSubview(phoneNumberView)
+        self.phoneNumberView.snp.makeConstraints { make in
+            make.height.equalTo(60)
+            make.top.equalTo(nickNameView.snp.bottom).offset(28)
+            make.leading.trailing.equalToSuperview().inset(15)
+        }
+    }
+    
+    private func setEmailView() {
+        self.view.addSubview(emailView)
+        self.emailView.snp.makeConstraints { make in
+            make.height.equalTo(60)
+            make.top.equalTo(phoneNumberView.snp.bottom).offset(28)
+            make.leading.trailing.equalToSuperview().inset(15)
+        }
+    }
+    
+    private func setAgreementView() {
+        self.view.addSubview(agreementView)
+        self.agreementView.snp.makeConstraints { make in
+            make.height.equalTo(44)
+            make.top.equalTo(emailView.snp.bottom).offset(40)
+            make.leading.trailing.equalToSuperview().inset(15)
+        }
+        
+        self.agreementView.addSubview(agreementButton)
+        self.agreementButton.snp.makeConstraints { make in
+            make.centerY.equalTo(self.agreementView)
+            make.leading.equalToSuperview()
+            make.width.equalTo(43)
+        }
+        
+        self.agreementView.addSubview(agreementLabel)
+        self.agreementLabel.snp.makeConstraints { make in
+            make.centerY.equalTo(self.agreementView)
+            make.leading.equalTo(self.agreementButton.snp.trailing).offset(7)
+            make.trailing.equalTo(self.agreementView.snp.trailing).offset(0)
+        }
+    }
+    
+    private func setCompleteButtonView() {
+        self.view.addSubview(completeButton)
+        self.completeButton.snp.makeConstraints { make in
+            make.top.equalTo(agreementView.snp.bottom).offset(50)
+            make.leading.trailing.equalToSuperview().inset(90)
+            make.height.equalTo(60)
+        }
+        
+        self.completeButton.addSubview(completeButtonLabel)
+        self.completeButtonLabel.snp.makeConstraints { make in
+            make.centerX.centerY.equalTo(completeButton)
```

Comment:
centerX, centerY는 center 하나로 합칠 수 있습니당
```suggestion
            make.center.equalToSuperview()
```

Reply:
요거 몰랐네여 참고할게여